stevebrownlee/BangazonWeb
# Removed unnecessary Using statements

As part of the cleanup efforts of the application before deployment, I've removed the unnecessary `using` statements in the `Program.cs` file.

Steps to test:
1. Switch to the `test` branch
2. Run `dotnet build` from the command line
3. Verify that project builds

Ref: https://trello.com/c/2kLuf9hp/31-remove-unnecessary-using-statements
